package.json

```diff
@@ -51,23 +51,23 @@
     ]
   },
   "dependencies": {
-    "aws-param-store": "^3.1.0",
-    "aws-sdk": "^2.580.0",
-    "debug": "^4.1.1"
+    "aws-param-store": "^3.2.0",
+    "aws-sdk": "^2.773.0",
+    "debug": "^4.2.0"
   },
   "devDependencies": {
-    "@types/aws-param-store": "^2.1.0",
+    "@types/aws-param-store": "2.1.1",
     "@types/debug": "^4.1.5",
-    "@types/jest": "^24.0.23",
-    "@types/node": "^12.12.14",
-    "eslint": "^6.7.1",
-    "eslint-config-neo": "^0.5.1",
-    "husky": "^3.1.0",
-    "jest": "^24.9.0",
-    "lint-staged": "^9.5.0",
-    "prettier": "^1.19.1",
-    "rimraf": "^3.0.0",
-    "ts-jest": "^24.2.0",
-    "typescript": "~3.7.2"
+    "@types/jest": "^26.0.14",
+    "@types/node": "^14.11.10",
+    "eslint": "^7.11.0",
+    "eslint-config-neo": "^0.5.2",
+    "husky": "^4.3.0",
+    "jest": "^26.5.3",
+    "lint-staged": "^10.4.2",
+    "prettier": "^2.1.2",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^26.4.1",
+    "typescript": "~4.0.3"
```

Comment:
Our ESLint config doesn't work with TypeScript 4 either so change to the latest version of 3.x.